lib/MusicBrainz/Server/Entity/WikipediaExtract.pm

```diff
@@ -24,13 +24,10 @@ has 'language' => (
     isa => 'Str',
 );
 
-sub url
```

Comment:
That was moved because `-convert_blessed_universally` doesn't call methods, right? Would it make sense to define a `TO_JSON` instead?

It's unclear if the other attributes in this class are still needed, or if they were only used by the `url` method and can now be removed.

Reply:
It depends whether `TO_JSON` is less expensive than converting blessed attributes.

* `title` is not needed as it was only used by the `url` method, thus could be removed.
* `content` is needed for display.
* `canonical` is not needed at all, thus can be removed.
* `language` is not used, but it could be appended in parenthesis next to _Wikipedia_ in case it differs from MB UI language.

Reply:
I guess they're not very different performance-wise. Removing the `url` method does mean we have to construct the url again everywhere `WikipediaExtract->new` is called, but apparently there is only one usage. :)

Reply:
Displaying language name can will wait for languages being cached with code.